patches/messagesButton.js

```diff
@@ -0,0 +1,21 @@
+function move() {
+    const messages = document.createElement("div")
+    messages.innerHTML = document.querySelector(".messages").innerHTML
+    messages.style.float = "right"
+    messages.style.padding = "20px"
+    messages.style.marginLeft = "auto"

```

Comment:
Animation of the unread messages counter is running each time I change a site

[Screencast from 2024-09-08 23-51-11.webm](https://github.com/user-attachments/assets/e139cb5b-d828-4d18-8b07-02214eda1f05)

I think we want to add `messges.querySelector(".MuiBadge-anchorOriginTopRightRectangle").style.translation = "none"` to fix that (if it's issue not related to code executing when it shouldn't)